src/pages/profile/profile.jsx

```diff
@@ -20,6 +21,7 @@ const Profile = () => {
 
   return (
     <div className="routingButtons">
+      <Header />
```

Comment:
Add the header to `<Layout />`

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/61508cbbb6209a5b4971fa235ad9991b4978da1a/src/components/layout/Layout.js#L9-L11